components/script/dom/document.rs

```diff
@@ -6666,7 +6666,7 @@ pub(crate) fn determine_policy_for_token(token: &str) -> ReferrerPolicy {
         "same-origin" => ReferrerPolicy::SameOrigin,
         "strict-origin" => ReferrerPolicy::StrictOrigin,
         "default" | "strict-origin-when-cross-origin" => ReferrerPolicy::StrictOriginWhenCrossOrigin,
-        "origin-when-cross-origin" => ReferrerPolicy::OriginWhenCrossOrigin,
+        "origin-when-cross-origin" | "origin-when-crossorigin" => ReferrerPolicy::OriginWhenCrossOrigin,
```

Comment:
I'm also wondering if I need to add this new value as an atom in `local_names.txt` in `web_atoms`?